server/controllers/match/placementRequests/occupancyViewController.ts

```diff
@@ -0,0 +1,23 @@
+import type { Request, Response, TypedRequestHandler } from 'express'
+import { PlacementRequestService } from '../../../services'
+
+interface NewRequest extends Request {
+  params: { id: string }
+  query: { premisesName: string }
```

Comment:
I think this page will be needing lots of information from the premises - so you'll be needing the id passing in and make API requests to get everything including the name.

Reply:
yep, thanks. I've got that in a local branch that gets the matching details data so that can follow if okay?

Reply:
actually, in adding the new `Continue` button (as per convo above) I needed to pass all this info now so done as part of this PR afterall. Thanks!